salt/base/salt.sls

```diff
@@ -35,23 +35,12 @@ remove_old_salt_repo:
 
 salt-repo:
   pkgrepo.managed:
-    - humanname: repo.saltstack.org
-    {% if grains["oscodename"] == "focal" %}
-    - name: deb https://archive.repo.saltproject.io/py3/ubuntu/20.04/{{ grains["osarch"] }}/archive/3004 focal main
-    - key_url: https://archive.repo.saltproject.io/py3/ubuntu/20.04/{{ grains["osarch"] }}/archive/3004/salt-archive-keyring.gpg
-    {% elif grains["oscodename"] == "jammy" %}
-    - name: deb [signed-by=/etc/apt/keyrings/salt-archive-keyring.gpg arch={{ grains["osarch"] }}] https://repo.saltproject.io/salt/py3/ubuntu/22.04/{{ grains["osarch"] }}/3007 jammy main
-    - key_url: https://repo.saltproject.io/salt/py3/ubuntu/22.04/{{ grains["osarch"] }}/SALT-PROJECT-GPG-PUBKEY-2023.gpg
+    {% if grains["oscodename"] in ["focal", "jammy", "noble"] %}
+    - name: deb [signed-by=/etc/apt/keyrings/salt-archive-keyring-2024.pgp arch={{ grains["osarch"] }}] https://packages.broadcom.com/artifactory/saltproject-deb/ stable main
```

Comment:
Should we be doing something to pin the version of salt here?

Reply:
[1050e3a](https://github.com/python/psf-salt/pull/524/commits/1050e3a0dd8cb0f3cf8407622c32d29a1adf6e6c)